### PR TITLE
Adding serializer extmod

### PR DESCRIPTION
**Pull request by onlyanegg**
### What does this PR do?
Adds the ability to use custom serializer modules through the external modules interface

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/22257

### Tests written?
Yes

### Commits signed with GPG?
No

### CODE REVIEW

File: salt/modules/saltutil.py [hunk 1]
```diff
@@ -737,6 +737,45 @@ def sync_utils(saltenv=None, refresh=True, extmod_whitelist=None, extmod_blackli
     return ret
 
 
+def sync_serializers(saltenv=None, refresh=True, extmod_whitelist=None, extmod_blacklist=None):
+    '''
+    .. versionadded:: ?
```
Comment (rallytime) [inline salt/modules/saltutil.py]: `Fluorine` :)

File: salt/runners/saltutil.py [hunk 2]
```diff
@@ -580,3 +581,29 @@ def sync_eauth_tokens(saltenv='base', extmod_whitelist=None, extmod_blacklist=No
     '''
     return salt.utils.extmods.sync(__opts__, 'tokens', saltenv=saltenv, extmod_whitelist=extmod_whitelist,
                                    extmod_blacklist=extmod_blacklist)[0]
+
+
+def sync_serializers(saltenv='base', extmod_whitelist=None, extmod_blacklist=None):
+    '''
+    .. versionadded::
```
Comment (rallytime) [inline salt/runners/saltutil.py]: `Fluorine` here, too.

**Review by rallytime — CHANGES_REQUESTED** [review]
Nicely done - just a couple of small changes are needed here. :)

There's a lint error on this as well, but it's related to the versionadded line.

**Comment by onlyanegg** [comment]
Thanks, @rallytime! I added those changes.

**Review by rallytime — APPROVED** [review]

**Pull request merged**